sonarts-core/tests/rules/noHardcodedSecretsRule/noHardcodedSecretsRule.lint.ts

```diff
@@ -0,0 +1,20 @@
+
+
+function foo() {
+  const password = "secret";
+//                 ^^^^^^^^ {{Review this potentially hardcoded secret.}}
```

Comment:
@saberduck I would prefer put in this test some simple literal (without "secret") so that it's explicit that issue is raised due to variable name